.github/workflows/php.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: composer install --prefer-dist --no-interaction --no-suggest
 
       - name: Run PHP linter
-        run: composer ecs
+        run: composer cs
```

Comment:
If you have changed the command values, don't forget to fix the makefile as well